src/python/grpcio/grpc/_cython/_cygrpc/completion_queue.pyx

```diff
@@ -74,7 +68,41 @@ cdef class CompletionQueue:
     with self.poll_condition:
       self.is_polling = False
       self.poll_condition.notify()
+    return self.interpret_event(event)
+
+  def pluck(self, records.OperationTag tag, records.Timespec deadline=None):
+    # Plucking a 'None' tag is equivalent to passing control to GRPC core until
+    # the deadline.
+    # TODO(atash) there's currently no way of getting a tag to pass to this
```

Comment:
I'm not sure I understand this comment, at least as far as its belonging at this point in the code. If there's no way to get a non-`None` tag to pass to this method, isn't that a problem for callers of the method, and maybe the comment should go outside the method instead of inside its body? Or am I misunderstanding what the comment is saying?